ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/privacy/methods/priv/PrivGetTransactionCount.java

```diff
@@ -51,7 +51,7 @@ public JsonRpcResponse doResponse(final JsonRpcRequestContext requestContext) {
     final Address address = requestContext.getRequiredParameter(0, Address.class);
     final String privacyGroupId = requestContext.getRequiredParameter(1, String.class);
 
-    final long nonce = privateTransactionHandler.getSenderNonce(address, privacyGroupId);
+    final long nonce = privateTransactionHandler.determineNonce(address, privacyGroupId);
```

Comment:
nit: I can't help but think the privateTransactionHandler should be a base-class member ... every Priv Json RPC seems to need it...

Reply:
They all do, not convinced we are gaining a lot by added a base class though as it will just passed into the constructor anyway.